assets/js/page-enhancements.js

```diff
@@ -144,6 +144,23 @@ window.pageEnhancements = (($, document) => {
     })
   }
 
+  const pushGtmBarcodeCount = () => {
+    const barcodesCreatedCount = $('#barcode-created-count').attr('data-value')
+    if (barcodesCreatedCount) {
+      window.dataLayer.push({
+        barcodesCreatedCount: barcodesCreatedCount,
+      })
+    }
+  }
+  const pushGtmScannedBarcodeNumber = () => {
```

Comment:
I know its part of the ACs, but I'm wondering what the value of pushing the scanned barcode number to GA is? What will they do with that info?
I'm also slightly uncomfortable about sending the barcode number, though I cant qualify it. I know the number will have been scanned at that point so cannot be reused; but something is not quite sitting right with me. Maybe understanding the use case as to why they want the scanned barcode number might alleviate my concerns ?